cfme/tests/cloud/test_providers.py

```diff
@@ -618,12 +646,63 @@ def test_regions_gov_azure(provider):
         setup: Check the region list when adding a Azure Provider.
         startsin: 5.7
     """
-    view = navigate_to(AzureProvider, "Add")
-    # prefill the provider type to enable regions dropdown
-    view.fill({'prov_type': provider.type.capitalize()})
-    available_regions = [opt.text for opt in view.region.all_options]
     # no government regions should be available by default
-    assert not any(reg for reg in available_regions if 'gov' in reg.lower())
+    assert not azure_has_available_gov_regions()
+
+
+@test_requirements.azure
+@pytest.mark.provider([AzureProvider], scope="function", selector=ONE)
+@pytest.mark.tier(2)
+def test_regions_all_azure(azure_available_regions_backend, azure_available_regions):
+    """
+    Need to validate the list of regions we show in the UI compared with
+    regions.rb  Recent additions include UK South
+    These really don"t change much, but you can use this test case id
+    inside bugzilla to set qe_test flag.
```

Comment:
Is this automating a BZ? If so, you should include that BZ ID as `automates` coverage

Reply:
I didn't find BZ that seems related to this test case :/